server/src/pages/shot/view.js

```diff
@@ -363,68 +363,63 @@ class Body extends React.Component {
     let favoriteShotButton = null;
     let trashOrFlagButton = null;
     let editButton = null;
-    const highlight = this.state.highlightEditButton
-      ? <div className="edit-highlight"
-          onClick={this.onClickEdit.bind(this)}
-          onMouseOver={this.onMouseOverHighlight.bind(this)}
-          onMouseOut={this.onMouseOutHighlight.bind(this)}></div>
-      : null;
-    const activeFavClass = this.props.expireTime ? "" : "is-fav";
-    const inactive = this.props.isFxaAuthenticated ? "" : "inactive";
-
-    favoriteShotButton = <div className="favorite-shot-button">
-      <Localized id="shotPagefavoriteButton" attrs={{title: true}}>
-        <button className={`nav-button ${inactive}`}
-          disabled={!this.props.isFxaAuthenticated}
-          onClick={this.onClickFavorite.bind(this)}>
-          <span className={`icon-favorite favorite ${activeFavClass}`} ></span>
-          <Localized id="shotPageFavorite">
-            <span className={`favorite-text favorite ${activeFavClass} `}>Favorite</span>
-          </Localized>
-        </button></Localized></div>;
-
-    const downloadButton = <div className="download-shot-button">
-      <Localized id="shotPageDownloadShot" attrs={{title: true}}>
-        <button className="nav-button icon-download" onClick={this.onClickDownload.bind(this)}
-          title="Download the shot image">
-          <Localized id="shotPageDownload">
-            <span>Download</span>
-          </Localized>
-        </button>
-      </Localized></div>;
-
-    const copyButton = <div className="copy-img-button" hidden={this.state.isClient && !this.state.canCopy}>
-      <Localized id="shotPageCopyButton" attrs={{title: true}}>
-        <button className="nav-button icon-copy transparent copy"
-          title="Copy image to clipboard"
-          onClick={this.onClickCopy.bind(this)}>
-          <Localized id="shotPageCopy">
-            <span>Copy Image</span>
-          </Localized>
-        </button>
-      </Localized>
-    </div>;
+    let downloadButton = null;
+    let copyButton = null;
 
     if (this.props.isOwner) {
+      const highlight = this.state.highlightEditButton
+        ? <div className="edit-highlight"
```

Comment:
Maybe we should replace this svg highlight with a separate highlighted-pen image (fine to do this in a followup bug)